docs/library/ppsspp.md

```diff
@@ -355,4 +355,4 @@ Settings with (Restart) means that core has to be closed for the new setting to
 - [Official PPSSPP Github Repository](https://github.com/hrydgard/ppsspp)
 - [Libretro PPSSPP Core info file](https://github.com/libretro/libretro-super/blob/master/dist/info/ppsspp_libretro.info)
 - [Report Libretro PPSSPP Core Issues Here](https://github.com/libretro/ppsspp/issues)
-- [Gameplay Videos](https://www.youtube.com/playlist?list=PLRbgg4gk_0IcjmP26m8-3JWNzYUXRjqWT)
\ No newline at end of file
+- [Gameplay Videos](https://www.youtube.com/playlist?list=PLRbgg4gk_0IcjmP26m8-3JWNzYUXRjqWT)
```

Comment:
Created the pull request from github dash. Possibly an added newline here or something. I can remove it if it's not wanted.